src/parser/xasl_generation.c

```diff
@@ -10482,6 +10516,41 @@ pt_to_rangelist_key (PARSER_CONTEXT * parser, PT_NODE ** term_exprs, int nterms,
 		  break;
 		}
 
+	      if (multi_col_pos[i] != -1)
+		{
+		  /* case of multi column term. case1 : value type, case2 : function type */
+		  if (pt_is_set_type (llim) && PT_IS_VALUE_NODE (llim))
```

Comment:
PT_IS_VALUE_NODE can be replaced with pt_is_value_node for readability but it's not important.

Reply:
i fix it